pkg/apigw/registry/registry.go

```diff
@@ -8,19 +8,22 @@ import (
 	"github.com/cortezaproject/corteza-server/pkg/apigw/filter"
 	"github.com/cortezaproject/corteza-server/pkg/apigw/filter/proxy"
 	"github.com/cortezaproject/corteza-server/pkg/apigw/types"
+	"github.com/cortezaproject/corteza-server/pkg/options"
 )
 
 type (
 	Registry struct {
-		h map[string]types.Handler
+		opts *options.ApigwOpt
+		h    map[string]types.Handler
 	}
 
 	secureStorageTodo struct{}
 )
 
-func NewRegistry() *Registry {
+func NewRegistry(opts *options.ApigwOpt) *Registry {
```

Comment:
Options should b (unwritten Corteza rule) passed to the functions as values (not refs).

Reply:
:+1: